resources/user-agents/browsers/uc-browser/uc-web-11.json

```diff
@@ -29,7 +29,6 @@
       "children": [
         {
           "match": "Mozilla/5.0 (#PLATFORM##DEVICE# Build/*) applewebkit* (*khtml*like*gecko*) Version/* UCBrowser/#MAJORVER#.#MINORVER#* U3/* Safari/*",
-          "engine": "U3",
```

Comment:
This engine is defined above on line 12, so isn't necessary in all of these children.